### PR TITLE
conda_install benchmark from dholth/conda-benchmarks

### PR DESCRIPTION
**Pull request by dholth**
### Description

Missed from earlier merge/combine.

### Checklist - did you ...

- [ ] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

File: asv.conf.json [hunk 1]
```diff
@@ -85,7 +85,8 @@
 
     "matrix": {
         "req": {
-            "conda-package-handling": ["2.0.0a4"],
+            "conda-package-handling": ["2.0.2"],
+            "conda-libmamba-solver":[],
```
Comment (dholth) [inline asv.conf.json:89]: means "install the latest conda-libmamba-solver in the test environment"

File: tests/test_benchmarks.py [hunk 2]
```diff
@@ -15,90 +15,180 @@
 except ImportError:
```
Comment (dholth) [inline tests/test_benchmarks.py]: these tests don't run on osx-arm64 and were written against old dependencies

File: download_files.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
         "CONDA_PKGS_DIRS": os.path.join('repos', 'main', subdir),
 }):
     try:
-        python_api.run_command("create", "-n", "fakeenv", "--download-only", "python", "mkl")
+        python_api.run_command("create", "-n", "fakeenv", "--download-only", "python=3.6", "mkl")
```
Comment (dholth) [inline download_files.py]: gives an idea of the age of the solver tests, they probably aren't worth the trouble.

**Review by kenodegard — APPROVED** [review]
I have no opinions on this PR. There are too many changes for me to make a meaningful code review pass.

Approving since this is a low-impact repo. 🚀

**Pull request merged**